core/embed/gfx/fonts/fonts.c

```diff
@@ -24,300 +24,127 @@
 #include "librust_fonts.h"
```

Comment:
We no longer need to include `librust_fonts.h`, and we can delete the file as well.

Reply:
d42cea3818e154945ac7649bb2031822192adddb